docs/consume-packages/configuring-nuget-behavior.md

```diff
@@ -84,7 +90,7 @@ nuget config -set repositoryPath=/home/packages -configfile $XDG_DATA_HOME/NuGet
 ```
 
 > [!Note]
-> In NuGet 3.4 and later you can use environment variables in any value, as in `repositoryPath=%PACKAGEHOME%` (Windows) and `repositoryPath=$PACKAGEHOME` (Mac/Linux).
+> In NuGet 3.4+ you can use environment variables in any value, as in `repositoryPath=%PACKAGEHOME%` (Windows) and `repositoryPath=$PACKAGEHOME` (Mac/Linux).
```

Comment:
I expect that "3.4+" will be understood by most English speakers around the world. But I'm wondering if it will be well understood by non-English speakers who have a beginner level of English. Also, I'm wondering what the automatic translation into different languages will be like and if all the languages that learn.microsoft.com supports uses/understand the same "3.4+" convention, particularly those languages that don't use the latin alphabet.

Perhaps it's safer to leave "3.4 or later"?